### PR TITLE
add runs-on image tag

### PR DESCRIPTION
**Pull request by aluon**
## Description

Add a new image tag `aptos-ubuntu-x64-stable` that points to the latest image built by our packer workflow

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## How Has This Been Tested?

This just adds a new image tag. I can test this when updating our other workflows to use the image tag

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: .github/workflows/build-runs-on-image.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Build runs-on image
+
+on:
+  workflow_dispatch:
```
Comment (rustielin) [inline .github/workflows/build-runs-on-image.yaml:4]: Eventually we should run this on a cadence/nightly

File: .github/workflows/build-runs-on-image.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: Build runs-on image
+
+on:
```
Comment (rustielin) [inline .github/workflows/build-runs-on-image.yaml:3]: as a quick hack to test this on your PR, you can probably add the `push` trigger on specifically your branch `aluon/runs-on-image-workflow` :) make sure to remove it after you've confirmed it works though, so you don't accidentally land it

This way you don't have to wait until landing in main to try `workflow_dispatch`

File: .github/workflows/build-runs-on-image.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
```
Comment (rustielin) [inline .github/workflows/build-runs-on-image.yaml:17]: I don't feel too strongly about this, but have you considered moving this (and the entire packer setup actually) to internal-ops?

**Review by rustielin — CHANGES_REQUESTED** [review]

File: .github/workflows/build-runs-on-image.yaml (hunk 3 above)
Comment (aluon) [reply .github/workflows/build-runs-on-image.yaml]: I pulled the workflow out to another PR to internal-ops

**Comment by aluon** [comment]
Closing this in favor of #14909

**Pull request closed without merging**